modules/dop/component-protocol/components/project-pipeline/myPage/tabsTable/filterContainer/customFilter/condition.go

```diff
@@ -61,18 +63,18 @@ func (p *CustomFilter) StatusCondition() *model.SelectCondition {
 	return condition
 }
 
-func (p *CustomFilter) MemberCondition(key string) (*model.SelectCondition, error) {
+func (p *CustomFilter) MemberCondition() (*model.SelectCondition, *model.SelectCondition, error) {
```

Comment:
make a new struct include executorCondition and creatorCondition, unless we cannot easy to understand what this func returning.

Reply:
done